### PR TITLE
Update rise-storage

### PR DESCRIPTION
**Pull request by donnapep**
* Update `rise-storage`.
* Remove `angular-sanitize` bower resolution since bower was complaining it wasn't needed.

### CODE REVIEW

**Comment by donnapep** [comment]
@rodrigopavezi I didn't end up making the same change for an image folder as I'm not sure it's necessary. I wouldn't think we'd want to interrupt the slideshow to display a message if one of the files in a folder fails to load. We would just proceed with showing the rest of the slideshow. If you're in agreement, please go ahead and review. Thx.

File: dist/components/rise-storage-v2/rise-storage.html [hunk 1]
```diff
@@ -1136,6 +1136,10 @@
       }
     },
 
+    getMetadata: function() {
+      this._makeStorageRequest();
```
Comment (rodrigopavezi) [inline dist/components/rise-storage-v2/rise-storage.html]: @donnapep I think it needs to get the latest component code. cheers

Comment (donnapep) [inline dist/components/rise-storage-v2/rise-storage.html]: @rodrigopavezi Oops. I think I committed before rise-storage was merged. All fixed now. Thx.

Comment (rodrigopavezi) [inline dist/components/rise-storage-v2/rise-storage.html]: @donnapep looks good 👍  cheers

**Comment by donnapep** [comment]
@stulees @rodrigopavezi The call to fetch metadata has been offloaded to `rise-storage`. I left the other minor refactoring changes I'd already made. Please review. Thx.

File: src/widget/image.js [hunk 2]
```diff
@@ -171,6 +171,10 @@ RiseVision.Image = (function (gadgets) {
         "event_details": "image load error",
         "file_url": url
       }, true);
+
+      if (_storage) {
+        _storage.getMetadata();
```
Comment (donnapep) [inline src/widget/image.js]: @rodrigopavezi I think this may be an old commit? I removed that `getMetadata` call entirely in this PR.

Comment (rodrigopavezi) [inline src/widget/image.js]: @donnapep yeap, 👍  cheers

**Pull request closed without merging**